package.json

```diff
@@ -84,6 +84,7 @@
     "eslint-plugin-flowtype": "2.18.2",
     "firefox-client": "0.3.0",
     "flow-bin": "0.32.0",
+    "git-hooks": "1.1.1",
```

Comment:
@kumar303 I've give this a try on Windows 7 and it seems that it is currently unable to work because `git-hooks` didn't take into account how `git` integrate hooks on windows (it seems that a `bash` shell is included with the git window and used to run the hooks scripts).

I've tried to use `git-hooks-win` (it looks like a fork of `git-hooks` which aims to work as a drop-in replacement of the `git-hooks` npm package) and it seems to be able to run these hooks correctly on both linux and windows 7 (I've not yet explicitly tried it on osx).